Html css session day 3

gealsanchez opened the pull request:
.github/workflows FOLDERS were CREATED workflows goes inside .github
linters.yml FILE was CREATED  inside workflows folder
linters.yml HTML & CSS configuration

1 "CONTAINER" DIV WAS CREATED
3 DIVs WERE CREATED INSIDE THE CONTAINER
5 CLASSES WERE CREATED "CONTAINER" , "ICON" , "NAME" , "PROFILE" , "LEFTMARGIN"

"container" , "icon" , "name" , "profile" , "leftmargin"  CLASSES WERE CREATED

linters.yml errors corrected

Optional Suggestions**

Update the document title Professional Portfolio GERSON SANCHEZ
Changed the width from px to % 
Font families was setup
geometry.png added
"navigation" , "menu" CLASSES WERE ADDED
hamburguer.png ICON line WAS ADDED
hamburguer.png WAS UPLOADED
"navigation" , "menu" CLASSES WERE DECLARED
Linters.yml errors
 13:11  heavy_multiplication_x  Expected single space before "{"   block-opening-brace-space-before
 22:12  heavy_multiplication_x  Expected a leading zero            number-leading-zero

Required change

Kindly make sure you chose one temple to the Figma and your design must stick to the template design you have chosen as much as possible (e.g. font, colors, images, text, margins) using the templates in Figma.

gealsanchez committed:
Create linters.yml

.github/workflows FOLDERS were CREATED workflows goes inside .github
linters.yml FILE was CREATED  inside workflows folder
linters.yml HTML & CSS configuration

gealsanchez committed:
Create index.html file

gealsanchez committed:
Create styles.css file

gealsanchez committed:
update index.html

1 "CONTAINER" DIV WAS CREATED
3 DIVs WERE CREATED INSIDE THE CONTAINER
5 CLASSES WERE CREATED "CONTAINER" , "ICON" , "NAME" , "PROFILE" , "LEFTMARGIN"

gealsanchez committed:
Update styles.css

"container" , "icon" , "name" , "profile" , "leftmargin"  CLASSES WERE CREATED

gealsanchez committed:
Update style.css

Changed the indentation

gealsanchez committed:
Update style.css

Expected indentation of 2 spaces

gealsanchez committed:
Update styles.css

Unexpected extra semicolon

gealsanchez committed:
Update styles.css

Added semicolon 8:2

gealsanchez committed:
Update styles.css

Unexpected extra semicolon   no-extra-semicolons

 10:1  ✖  Unexpected extra semicolon   no-extra-semicolons
 16:1  ✖  Unexpected extra semicolon   no-extra-semicolons
 25:1  ✖  Unexpected extra semicolon   no-extra-semicolons
 38:1  ✖  Unexpected extra semicolon   no-extra-semicolons

gealsanchez committed:
Update styles.css

Changed the "leftmargin" & "icon" classes indentation let them in 1 line not in 3

gealsanchez committed:
Update styles.css

No space between classes

gealsanchez committed:
Update styles.css

Missed semicolon 11:21

gealsanchez committed:
Update styles.css

11:21 Missed semicolon

gealsanchez committed:
Update styles.css

  7:15 Expected a trailing semicolon declaration-block-trailing-semicolon
 10:1 Expected empty line before rule rule-empty-line-before              
 16:1 Expected empty line before rule rule-empty-line-before

gealsanchez committed:
Update styles.css

 16:2 Unexpected whitespace at end of line no-eol-whitespace                    
 17:1 Unexpected empty line before closing brace block-closing-brace-empty-line-before
 18:1 Expected empty line before rule rule-empty-line-before

gealsanchez committed:
Update style.css

 17:1 Expected empty line before rule rule-empty-line-before       
 18:2 Unexpected whitespace at end of line no-eol-whitespace            
 19:3 Unexpected empty line before declaration declaration-empty-line-before

gealsanchez committed:
Update README.md

gealsanchez committed:
Update README.md

Update License Link
[MIT license](https://github.com/gealsanchez/portfolio/blob/htmlCSS-Session-Day-3/LICENSE)

mgmediaweb commented on index.html:
```diff
@@ -0,0 +1,25 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
```
Kindly, I suggest you put a descriptive title to your website

mgmediaweb replied:
Good job following the advice of the previous reviewer

mgmediaweb commented on index.html:
```diff
@@ -0,0 +1,25 @@
+
+    <div class="container">
+        <div><img class="icon" src="geometry.png" alt=""></div>
+        <div class="name leftmargin">Gerson Sanchez</div>
```
Kindly, I suggest you use an H1 tag for the main header

mgmediaweb replied:
Good job following the advice of the previous reviewer

mgmediaweb commented on index.html:
```diff
@@ -0,0 +1,25 @@
+    <div class="container">
+        <div><img class="icon" src="geometry.png" alt=""></div>
+        <div class="name leftmargin">Gerson Sanchez</div>
+        <div class="profile leftmargin">Hello! I am a software developer! I can help you build a product, feature or
```
I kindly suggest using a "P" tag for traditional information

mgmediaweb replied:
Good job following the advice of the previous reviewer

mgmediaweb commented on styles.css:
```diff
@@ -0,0 +1,24 @@
+  flex-direction: column;
+  flex-wrap: wrap;
+  border: 10px solid blue;
+  width: 320px;
```
Kindly, I suggest you not to use fixed widths (it is preferable to use percentages) so that your website adapts to any size of mobile device.

mgmediaweb commented on index.html:
```diff
@@ -0,0 +1,25 @@
+<body>
+
+    <div class="container">
+        <div><img class="icon" src="geometry.png" alt=""></div>
```
Kindly, I suggest you use a HEADER tag, and within it a NAV tag to contain the "hamburger" menu

mgmediaweb replied:
Kindly, I suggest you check this image link because the image does not exist

mgmediaweb replied:
Good job following the advice of the previous reviewer

mgmediaweb reviewed (changes requested):
Hi @gealsanchez , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![work2](https://user-images.githubusercontent.com/41912912/191652162-7a03be6e-fa84-4c1b-a8ff-ee11357d204b.gif)

## Highlights 💯 

- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 
- Clean and tidy code. ✔️ 

## Required Changes ♻️

*** Kindly, I suggest you work a little more on your design, currently I can't tell which template you are basing on Figma

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gealsanchez committed:
Update index.html

Update the document title Professional Portfolio GERSON SANCHEZ

gealsanchez committed:
Update styles.css

Changed the width from px to % 
Font families was setup

gealsanchez committed:
Add files via upload

geometry.png added

gealsanchez committed:
Update index.html

"navigation" , "menu" CLASSES WERE ADDED
hamburguer.png ICON line WAS ADDED

gealsanchez committed:
Add files via upload

hamburguer.png WAS UPLOADED

gealsanchez committed:
Update styles.css

"navigation" , "menu" CLASSES WERE DECLARED

gealsanchez committed:
Update styles.css

 13:11  ✖  Expected single space before "{"   block-opening-brace-space-before
 22:12  ✖  Expected a leading zero            number-leading-zero

MaganaAsiati commented on index.html:
```diff
@@ -0,0 +1,25 @@
+<body>
+
+    <div class="container">
+        <div><img class="icon" src="geometry.png" alt=""></div>
```
- [ ] Kindly make sure the container of this section is a `header `tag. 
Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. :grinning:  :grinning:  :grinning: 

- [ ] The logo (if the design has one) is on the left and has a link (`a` tag).
   * This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.  :grinning:  :grinning:  :grinning: 

- [ ] The hamburger button is on the right and has a `nav` tag.
   * The `nav` tag is for semantics reasons, and the right position for convention and user experience for mobile devices.  :grinning:  :grinning:  :grinning: 

 - check the example below :+1: 

```
<header>
        <a href="index.html">
          <img src="images/logo.png" alt="Logo" />
        </a>

        <nav class="hamburger-menu">
          <div class="bar-top"></div>
          <div class="bar-middle"></div>
          <div class="bar-bottom"></div>
        </nav>
    </header>
```

MaganaAsiati reviewed (changes requested):
# STATUS: Required Changes ♻️♻️♻️ 

Hi @gealsanchez 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![AlonzoLeroneThisCloseGIF](https://user-images.githubusercontent.com/50941074/161814298-b87179c3-315f-4207-ac3b-1d3e584d506e.gif)

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️

- [ ] Kindly make sure you chose one temple to the Figma and your design must stick to the template design you have chosen as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). :grinning: :grinning: :grinning: 

### For example if you chose a template  one your design will look at the picture below 
     
## the toolbar (or header):
    
   ![image](https://user-images.githubusercontent.com/95297251/191835039-3d08030d-05a5-407b-9ee4-dfa839e7c52e.png)

## the headline section (right after the header):
    
![image](https://user-images.githubusercontent.com/95297251/191835527-5299e7d7-1fa3-4d4e-919f-6840abb7cc0c.png)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

gealsanchez committed:
Update styles.css

Kindly make sure you chose one temple to the Figma and your design must stick to the template design you have chosen as much as possible (e.g. font, colors, images, text, margins) using the templates in Figma.

gealsanchez committed:
Update index.html

 Kindly make sure you chose one temple to the Figma and your design must stick to the template design you have chosen as much as possible (e.g. font, colors, images, text, margins) using the templates in Figma.

gealsanchez committed:
Update styles.css

 23:12  ✖  Expected a leading zero                    number-leading-zero          
 46:3   ✖  Unexpected empty line before declaration   declaration-empty-line-before

gealsanchez committed:
Update styles.css

width: 50%

gealsanchez committed:
Update styles.css

Placed the right image in the right bottom side

gealsanchez committed:
Update index.html

Added a div to align the social media links and the image of the left side

gealsanchez committed:
Update style.css

Duplicated lines deleted

gealsanchez committed:
Update styles.css

 54:8  ✖  Expected single space before "{"   block-opening-brace-space-before

mgmediaweb reviewed (changes requested):
Hi @gealsanchez , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![almost5](https://user-images.githubusercontent.com/41912912/191888805-ed5285a0-5dd1-4670-b868-9ff0cccd953c.gif)

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️

- Kindly, I suggest you check a white border and a horizontal scroll that makes your website not look very good, please check what element produces this and adjust it
- Kindly, I suggest you revise the background image and social icons, so that it looks similar to the original Figma design.

![image](https://user-images.githubusercontent.com/41912912/191888574-530bd83d-ba3f-4401-b95a-53f02836d708.png)


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gealsanchez committed:
Add files via upload

Social media and images were uploaded

gealsanchez committed:
Update styles.css

body margin 0%

gealsanchez commented:
I forgot to upload the images my mistake sorry

gealsanchez committed:
Delete geometry.png

No more needed

Nyame-Wolf commented on styles.css:
```diff
@@ -0,0 +1,85 @@
+  display: flex;
+  flex-direction: column;
+  border-radius: 2%;
+  width: 375px;
```
[OPTIONAL]I suggest getting rid of this fixed width. The app looks good but can serve only devices that have a minimum width of 375 px while its meant to serve devices between 768px and 375px

gealsanchez replied:
Hi @Nyame-Wolf how do you describe that behavior, dynamic width?

Nyame-Wolf commented on index.html:
```diff
@@ -0,0 +1,45 @@
+    <div class="container">
+        <header>
+            <nav class="navigation">
+                <p>John Doe</p>
```
[OPTIONAL] I suggest making this logo a link element `<a>` instead of a paragraph. 

- The logo (if the design has one) is on the left and has a link (a tag).
This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

Nyame-Wolf commented on index.html:
```diff
@@ -0,0 +1,45 @@
+<body>
+
+    <div class="container">
+        <header>
```
[OPTIONAL] The margin between your header elements is not similar to the figma design

- [ ] Your design
<img width="679" alt="Screenshot 2022-09-23 at 15 59 52" src="https://user-images.githubusercontent.com/48546992/191966426-3670a8df-8814-4e47-b19a-af2cbe72882e.png">

- [ ] Figma design
<img width="664" alt="Screenshot 2022-09-23 at 15 59 12" src="https://user-images.githubusercontent.com/48546992/191966473-930b02bc-6621-492d-b691-dc2e8425116e.png">


- [ ]  Please note that your color and fonts also do not match the figma design. Please fix this.

Nyame-Wolf reviewed (approved):
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @gealsanchez :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![](https://media.giphy.com/media/l4q7TIW8nEZYOJUf6/giphy.gif)


## Highlights :sparkles:

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
